images/deployer/Dockerfile

```diff
@@ -9,4 +9,11 @@
 #
 FROM openshift/origin
 
+RUN yum install -y nfs-utils sudo
+
+# Edit sudoers file
+# To avoid error: sudo: sorry, you must have a tty to run sudo
+RUN sed -i -e "s/Defaults    requiretty.*/ #Defaults    requiretty/g" /etc/sudoers
```

Comment:
Agreed, I don't want these use cases to continue to overlap. If this change requires that, then this change is not going to make 3.1